label-application-rule-config-service-api/src/main/proto/org/hypertrace/label/application/rule/config/service/v1/label_application_rule.proto

```diff
@@ -0,0 +1,98 @@
+syntax = "proto3";
+
+package org.hypertrace.label.application.rule.config.service.v1;
+
+option java_multiple_files = true;
+
+message LabelApplicationRule {
+  string id = 1;
+  LabelApplicationRule data = 2;
+}
+
+message CreateLabelApplicationRule {
+  LabelApplicationRule data = 2;
+}
+
+message LabelApplicationRuleData {
+  string name = 1;
+  string description = 2;
+  MatchingCriteria matching_criteria = 3;
+  ApplicationCriteria application_criteria = 4;
+
+  message MatchingCriteria {
+    Type type = 1;
+    repeated Condition conditions = 2;
+
+    enum Type {
+      TYPE_UNSPECIFIED = 0;
+      TYPE_SPAN = 1;
+      TYPE_TRACE = 2;
+    }
+
+    message Condition {
+      string name = 1;
```

Comment:
What's name?

Reply:
Condition name like `Condition 1` 
Depends how it will be displayed in UI.

Reply:
I would expect we'd display the condition fields themselves - I don't think there's any need to name a sub-object of a rule.

Reply:
Removed